src/sentry/integrations/vsts/integration.py

```diff
@@ -81,6 +36,31 @@ def get_client(self):
             raise ValueError('Identity missing access token')
         return VstsApiClient(access_token)
 
+    def get_project_config(self):
+        client = self.get_client()
+        projects = client.get_projects(self.model.metadata['domain_name'])
+        project_choices = [(project['id'], project['name']) for project in projects['value']]
+        default_project = self.org_integration.config.get('default_project')
+
+        # TODO(LB): This is only saving the id. That might work... but really I'd
+        # want to save the name. Help? Maybe name and id?
+        return [
+            {
+                'name': 'default_project',
+                'type': 'choice',
+                'allowEmpty': True,
+                'required': True,
+                'choices': project_choices,
+                'initial': (default_project['id'], default_project['name']) if default_project is not None else ('', ''),
```

Comment:
Is there no simple way to save both the name and the id from this?

Reply:
essentially I feel like it makes sense to save something like this:
```python
'default_project': {
         'name': project['name'],
          'id': project['id'],
   }
```

Reply:
we've just saved id in the past and relied on the API to know what the name is (JIRA has the same problem)

Reply:
Yeah, unless we have to worry about performance we can just store the project ID for now and anywhere we need to display the project name we'll query the API.